perfkitbenchmarker/vm_util.py

```diff
@@ -362,23 +373,33 @@ def IssueCommand(cmd, force_info_log=False, suppress_warning=False):
   return stdout, stderr, process.returncode
 
 
-def IssueBackgroundCommand(cmd, stdout_path, stderr_path):
+def IssueBackgroundCommand(cmd, stdout_path, stderr_path, env=None):
   """Run the provided command once in the background.
 
   Args:
     cmd: Command to be run, as expected by subprocess.Popen.
     stdout_path: Redirect stdout here. Overwritten.
     stderr_path: Redirect stderr here. Overwritten.
+    env: A dict of key/value strings, such as is given to the subprocess.Popen()
+        constructor, that contains environment variables to be injected.
   """
+  if env is not None:
+    cmd_env = env
+  else:
+    cmd_env = os.environ.copy()
```

Comment:
I think passing `env` directly should be fine - `None` is the default value; as best I can tell the Popen constructor selects `os.execvp` or [`os.execvpe`](https://docs.python.org/2/library/os.html#os.execvpe) based on whether `env` is None.